backend/tests/owasp/scraper_test.py

```diff
@@ -108,16 +108,6 @@ def test_initialization_not_found(self, mock_session):
 
         assert scraper.page_tree is None
 
-    def test_get_leaders_no_leaders(self, mock_session):
```

Comment:
I believe this test needs to be refactored too based on the new logic and the code location.